docs/report/code-listing.tex

```diff
@@ -0,0 +1,266 @@
+\section*{Code listing for the software}

```

Comment:
Yay!
Can you do an overleaf project and add me as a collaborator ? ❤️

Reply:
(The line has nothing to do with the review but had to put it on a line lol)

Reply:
@guilyx Sure, I sent you the invite on overleaf